.eslintignore

```diff
@@ -3,3 +3,5 @@
 
 # Ignore WASM generated files:
 web/packages/teleport/src/ironrdp/pkg
+
+build
```

Comment:
Did you mean to ignore `web/packages/teleterm/build`?

Reply:
Eslint ignored the Webpack builds probably because of this:

https://github.com/gravitational/teleport/blob/218d608f835b8b15c2e8dca2a3542652d470ed4c/web/packages/build/.eslintrc.js#L37

And the files prepared by Webpack were in `web/packages/teleterm/build/app/dist`.